properties.schema

```diff
@@ -148,6 +148,15 @@
                         "inputType": { "type": "Select", "options": ["document", "media", "link"]},
                         "validators": ["required"]
                       },
+                      "forceDownload": {
```

Comment:
the standard we usually try to follow is that configuration properties start with an underscore, would you mind changing this to `_forceDownload` to keep with the convention?